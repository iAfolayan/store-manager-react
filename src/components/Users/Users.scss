table.table {
  width: 97%;
  margin: 70px auto
}