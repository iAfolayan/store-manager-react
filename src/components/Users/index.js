import React, { Component, Fragment } from 'react';
import { NavLink } from 'react-router-dom';
import './Users.scss';

class Users extends Component {
  render() {
    return (
    <Fragment>
        <h5 className="ml-3 text-primary">Users</h5>
        <hr className="mr-5" />
      <div class="table-responsive-sm">
        <table class="table table-bordered table-hover">
          <thead>
            <tr>
              <th scope="col">#</th>
              <th scope="col">Staff ID</th>
              <th scope="col">FUllname</th>
              <th scope="col">phone number</th>
              <th scope="col">ACTION</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <th scope="row">1</th>
              <td>SM001</td>
              <td>Mark Adeniran</td>
              <td>+2348032167911</td>
              <td>
                <NavLink to="edit">Edit</NavLink> | Delete</td>
            </tr>
            <tr>
              <th scope="row">2</th>
              <td>SM001</td>
              <td>Mark Adeniran</td>
              <td>+2348032167911</td>
              <td>
                <NavLink to="edit">Edit</NavLink> | Delete</td>
            </tr>
            <tr>
              <th scope="row">3</th>
              <td>SM001</td>
              <td>Mark Adeniran</td>
              <td>+2348032167911</td>
              <td>
                <NavLink to="edit">Edit</NavLink> | Delete</td>
            </tr>
          </tbody>
        </table>
      </div>
    </Fragment>
    );
  }
}

export default Users;