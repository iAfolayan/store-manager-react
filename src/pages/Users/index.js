import React, { Fragment } from 'react';
import Navigation from '../../components/Navbar';
import Users from '../../components/Users';
import Footer from '../../components/Footer';

const UsersPage = () => {
  return (
    <Fragment>
      <div className="form-group row">
        <div className="col-sm-12 col-md-2">
          <Navigation />
        </div>
        <div className="col-sm-12 col-md-10">
          <Users />
          <Footer footerClassName="footer inner" />
        </div>
      </div>
    </Fragment>
  );
}

export default UsersPage;
