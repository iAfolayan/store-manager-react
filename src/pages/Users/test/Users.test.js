import React from 'react';
import { shallow } from 'enzyme';
import Users from '..';

describe('User page', () => {
  test('should rendered user page without crashed', () => {
    const wrapper = shallow(<Users />);
    expect(wrapper).toMatchSnapshot();
  });
});