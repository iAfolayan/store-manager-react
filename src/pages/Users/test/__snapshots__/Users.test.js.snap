// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User page should rendered user page without crashed 1`] = `
<Fragment>
  <div
    className="form-group row"
  >
    <div
      className="col-sm-12 col-md-2"
    >
      <Navigation />
    </div>
    <div
      className="col-sm-12 col-md-10"
    >
      <Connect(Users) />
      <Footer
        footerClassName="footer inner"
      />
    </div>
  </div>
</Fragment>
`;
